synapse/handlers/oidc.py

```diff
@@ -1230,6 +1232,7 @@ def __init__(self, config: JinjaOidcMappingConfig):
     @staticmethod
     def parse_config(config: dict) -> JinjaOidcMappingConfig:
         subject_claim = config.get("subject_claim", "sub")
+        picture_claim = config.get("picture_claim", "picture")
```

Comment:
I have documented `picture_claim` in `docs/usage/configuration/config_documentation.md`. I couldn't find a good place to document anything in `docs/openid.md` since that involves examples of supported OIDC providers and I don't know if they are following convention for it. Most likely they are but I wasn't able to easily find them upon searching.

The ability to override them, if needed, is now documented though, so maybe its good enough? What do you think?

Reply:
Agreed. We'll let others update that config if needed.